coldfront/plugins/qumulo/hooks.py

```diff
@@ -0,0 +1,43 @@
+import textwrap
+
+from django.template.loader import render_to_string
+
+from coldfront.config.env import ENV
+from coldfront.core.utils.mail import (
+    EMAIL_ENABLED,
+    EMAIL_SENDER,
+    email_template_context,
+    send_acl_reset_email,
+)
+
+def acl_reset_complete_hook(task_object):
+    if EMAIL_ENABLED:
+        send_acl_reset_email(task_object)
+    else:
```

Comment:
I'm seeing a lot of code duplication here between here and `send_acl_reset_email`.  Easier would be to just get rid of the whole else statement/logging bit here.  You can always send calls to send an email and it won't send if `EMAIL_ENABLED` isn't True.  If we're insistent on writing emails to a directory, it should reuse all this context code and not duplicate it.